src/3d/qgs3dmapsettings.h

```diff
@@ -96,6 +97,8 @@ class _3D_EXPORT Qgs3DMapSettings : public QObject, public QgsTemporalRangeObjec
     void setCrs( const QgsCoordinateReferenceSystem &crs );
     //! Returns coordinate reference system used in the 3D scene
     QgsCoordinateReferenceSystem crs() const { return mCrs; }
+    //! Returns coordinate reference system used in the 3D scene
+    const QgsCoordinateReferenceSystem *crsConstPtr() const { return &mCrs; }
```

Comment:
QgsCoordinateReferenceSystem objects are implicitly shared -- they should always be used by value as a consequence

Reply:
reverted